src/data/faq.json

```diff
@@ -316,7 +316,15 @@
                         "anchor": "minimum_requirements",
                         "active": true,
                         "textblock": [
-                            "<b>UPDATE</b><br/><a href='https://www.coronawarn.app/en/blog/2020-12-15-ios-12-5' target='_blank' rel='noopener noreferrer'>Telekom and SAP to make Corona Warn app available for iOS 12.5.</a><hr/>",
+
+                            "<b>UPDATE 8th February 2021</b><br/>As of February 5, 2021, only the Corona-Warn-App (CWA) versions 1.5 and later are supported. For this reason, users with older versions will receive a notification to update their app accordingly. This applies to both iOS and Android. At the same time, however, certain minimum requirements apply to the operating system: Android at least 6.0 and iOS at least 13.7.",
+                            "iOS users who have previously used iOS 13.5, for example, must now update the iOS operating system to at least iOS 13.7 in order to continue using the CWA. Only then the technically correct operation of the app can be ensured.",
+                            "<ul><li>Please update your iOS version first. Older smartphones, such as the iPhone SE, 6S or 6S Plus, also support the latest iOS operating system. This allows you to upgrade your iPhone to at least iOS 13.7. For security reasons, however, you should keep your iPhone up-to-date, as Apple also regularly fixes minor security vulnerabilities with the updates. You can do this on your iPhone under Settings - General - Software Update.</li><li>Subsequently, please update the Corona-Warn-App. The Corona-Warn-App must also be installed at least in version 1.5. Version 1.11 is already available.</li></ul>",
+                            "Starting with CWA version 1.12, the older iPhone models iPhone 5s, iPhone 6, iPhone 6 Plus are also supported. These devices must be upgraded to iOS version 12.5 or 12.5.1.",

```

Comment:
iPhone 5s, iPhone 6, iPhone 6 Plus => iPhone 5s, iPhone 6 and iPhone 6 Plus

Reply:
ok